### DISCUSSION TITLE
Adapting VoucherVision for Mammals

### DISCUSSION THREAD

**Discussion by kendallfitzgerald** [Gene-Weaver/VoucherVision#30] [General Info & Subgroups]
Hello all! I'M Kendall, I am currently working at FMNH under the RANGES grant to digitize trait data for western North American mammals. I've been working a bit with @Gene-Weaver to adapt Voucher Vision for mammals collection, specifically for mammal skin tags. Thought I would start a discussion thread here in case anyone else in interested in trying this out, has questions or suggestions, etc. 

Basically, as opposed to VoucherVision with plants there are other fields we need the AI tool to collect specifically for mammals (e.g. measurements like TL, TV, HF, E etc.) and any notes for habitat or reproduction on the tag. I have a created a YAML for mammal skin tags using VoucherVision. I am testing that out now on the public demo of VoucherVision. So far it is able to pick up the measurements in the correct order but this will likely need more work and troubleshooting. One other issue is that skin tags have multiple sides,  so we need to collage the photographs of front and back of the skin tag (or multiple skin tags) to get all the information for one catalog number.  Currently thinking through things like that . . . 

Send me a message here if you're interested in trying it out :) I have a document of notes/introduction material I can share as well.

**Comment by Gene-Weaver** [comment]
We are adding a new label collage tool to build the collage from multiple image files. See #32

**Comment by kendallfitzgerald** [comment]
I've taken images of our skin tags to use in VoucherVision - the current workflow is to take jpg images (keeping under 999kb file size), cropping the images to include just the tags and renaming the files with underscores. Example here with the catalogue number 1234: FMNH_1234_a, FMNH_1234_b, FMNH_1234_c ,etc. Our workflow at the moment uses these underscores, so if you want to try this our don't use other annotations like periods, parentheses, commas or any other, stick to underscores. This can be changed/adapted later though for different workflows.
I can't attach my current YAML instruction file here but let me know if you want to try it out! Going to try to tune / train based on a 1400 specimen data set.
